### ISSUE TITLE
Feature request: Conform to OS/browser native dark/light mode setting

### ISSUE THREAD

**Issue by Quenz** [deanoemcke/thegreatsuspender#887]
* The Great Suspender v7.0.109
 * Google Chrome 73.0.3683.75 (Official Build) (64-bit)
 * macOS Mojave 10.14.3

Chrome's UI now conforms to the light/dark appearance setting in macOS.

[From Google Developers](https://developers.google.com/web/updates/2019/03/nic73):

> Dark mode for Chrome's UI is now supported on Mac, and Windows support is on the way. In addition, there's work on a CSS media query: [`prefers-color-scheme`](https://drafts.csswg.org/mediaqueries-5/#prefers-color-scheme), that can be used to detect if the user has requested the system use a light or dark color theme. The tracking bug for this is **Add support for CSS `prefers-color-scheme` media feature** For [Chrome](https://bugs.chromium.org/p/chromium/issues/detail?id=889087), and [Firefox](https://bugzilla.mozilla.org/show_bug.cgi?id=1494034).

I'm hoping this could be extended, so that the suspended tab page automatically changes depending on the native setting.

I'm not a developer, but Perhaps the CSS thing they mention could be used on the suspended tab page, since it is an HTML page?

**Comment by liamjohnston** [comment]
@Quenz yes, I've been thinking we should add this. Keen to keep new non-critical features out of the upcoming next release, but maybe I'll make a PR next time I get the chance 😊

I'm thinking it should be a third option in the theme dropdown, like "system setting" (or similar). We'd make it the default for those who currently have the default "light" set.

@deanoemcke Sound good?

**Comment by deanoemcke** [comment]
Lets put it in this release! :P

**Comment by liamjohnston** [comment]
@Quenz 
PR is here: https://github.com/deanoemcke/thegreatsuspender/pull/888

We're putting this on ice until Chrome ~v75 is out. 

Chrome v73 only adds support for Chrome itself to use on its tab bar etc, and doesn't provide access to the CSS media query (or JS matchMedia).

**Comment by piotr-chylarecki** [comment]
Guys, 
do you have plans to bring this? (When?) 

Switching the mode manually twice every day is nuts.

**Comment by thundercrypt** [comment]
Please implement this... It's the only thing holding me back from implementing auto switch.

**Comment by yorhodes** [comment]
I would be happy to implement this for one of the forks
cc @aciidic

**Comment by aciidic** [comment]
> I would be happy to implement this for one of the forks
> cc @aciidic

Sorry late reply, but I do check in on the fork occasionally and happy to accept pull requests
